packages/api/src/external/fhir-to-cda-converter/connector.ts

```diff
@@ -2,11 +2,10 @@ import { Bundle } from "../../routes/medical/schemas/fhir";
 
 export type FhirToCdaConverterRequest = {
   cxId: string;
-  patientId: string;
-  docId: string;
+  toSplit: boolean;
```

Comment:
Can you add a TSDoc here explaining what this is for, please? I suggest we use a more descriptive name, `toSplit` doesn't say much. Maybe `splitCompositions`?

Reply:
sounds good